src/Bookings.js

```diff
@@ -1,18 +1,55 @@
-import React from "react";
+import React, { useState } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.jsx";
+import FakeBookings from "./data/fakeBookings.json";
+import { useEffect } from "react";
 
 const Bookings = () => {
-  const search = searchVal => {
-    console.info("TO DO!", searchVal);
+  //console.log(useEffect);
+  const search = (searchVal) => {
+    const filterBookings = bookings.filter((booking) => {
+      return (
+        booking.firstName.toLowerCase().includes(searchVal) ||
+        booking.surname.toLowerCase().includes(searchVal)
+      );
+    });
+    setBookings(filterBookings);
   };
+  const [bookings, setBookings] = useState([]);
+
+  const [loading, setLoading] = useState(true);
+
+  const [error, setError] = useState(null);
```

Comment:
Best to keep your hooks at the very top of the file, so these three `useState` hooks would be better defined above the `search` function.